experimental/packages/opentelemetry-instrumentation-fetch/test/fetch.test.ts

```diff
@@ -210,12 +210,21 @@ describe('fetch', () => {
     sinon.stub(core.otperformance, 'now').callsFake(() => fakeNow);
 
     function fakeFetch(input: RequestInfo | Request, init: RequestInit = {}) {
+      // Once upon a time, there was a bug (#2411), causing a `Request` object
+      // to be incorrectly passed to `fetch()` as the second argument
+      assert.ok(!(init instanceof Request));
+
       return new Promise((resolve, reject) => {
-        const response: any = {
-          args: {},
-          url: fileUrl,
-        };
-        response.headers = Object.assign({}, init.headers);
+        const responseInit: ResponseInit = {};
+
+        // This is the default response body expected by the few tests that cares
+        let responseBody = JSON.stringify({
+          isServerResponse: true,
```

Comment:
`args: {}` in the diff is only used by a single test, and all it does with it is `assert.deepEquals(parsedResponseBody.args, {})`. Maybe it used to do more, but I inferred that these days the only purpose it serves is "did I get the mock response I expected", and so I replaced that with this field.